<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPcForm</class>
 <widget class="QDialog" name="AddPcForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PC Form</string>
  </property>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>292</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="modelLabel">
      <property name="text">
       <string>Model</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="modelLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="manufacturerLabel">
      <property name="text">
       <string>Manufacturer</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="manufacturerLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="photoUrlLabel">
      <property name="text">
       <string>Photo-Url</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="photoUrlLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="descriptionLabel">
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="descriptionLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="priceLabel">
      <property name="text">
       <string>Price</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="priceLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="ramLabel">
      <property name="text">
       <string>RAM</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="ramLineEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="cpuLabel">
      <property name="text">
       <string>CPU</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="cpuLineEdit"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="diskTypeLabel">
      <property name="text">
       <string>Disk Type</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="diskTypeLineEdit"/>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="diskSpaceLabel">
      <property name="text">
       <string>Disk Space</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="diskSpaceLineEdit"/>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="gPULabel">
      <property name="text">
       <string>GPU</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="gPULineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>320</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
